components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -3314,7 +3314,7 @@ def __init__(self, errorResult, arguments, argsPre, returnType,
         needsCx = needCx(returnType, (a for (a, _) in arguments), True)
 
         if "cx" not in argsPre and needsCx:
-            args.prepend(CGGeneric("cx"))
+            args.prepend(CGGeneric("JSContext::from_ptr(cx)"))
```

Comment:
I suspect that this will cause problems in the short term.

Reply:
Hmm did not we need to adjust this line? i.e.:

> - [ ] adjust generated code to pass JSContext::from_ptr(cx) when necessary (here)